templates/robotics/index.html

```diff
@@ -235,7 +237,8 @@ <h3 class="p-heading--2">Snapcraft</h3>
         <p>Snapcraft is the command-line tool for packaging your software as a Snap. Snaps are containerised software packages designed for embedded applications, with increased performance compared to <a href="/engage/dockerandros">other technologies</a>. Snaps give you all the interfaces you need to bundle your dependencies in one package, while providing secure interfaces to access the host disc and privileged resources.</p>
 
         <p>
-          <a href="https://snapcraft.io/docs/robotics" class="p-button--positive">Learn more about Snapcraft</a>
+          <a class="p-button--positive js-invoke-modal" href="/robotics/docs"> Explore documentation</a>
```

Comment:
It seems that the link is different here:
```suggestion
          <a class="p-button--positive js-invoke-modal" href="/robotics/docs/how-to-guides#heading--snapcraft"> Explore documentation</a>
```